x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_router.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+
+import { getEngineBreadcrumbs } from '../engine';
+
+import { CrawlerLanding } from './crawler_landing';
+
+export const CrawlerRouter: React.FC = () => {
+  return (
+    <Switch>
+      <Route>
+        <SetPageChrome trail={getEngineBreadcrumbs()} />
```

Comment:
This needs a Crawler breadcrumb!

```js
import { CRAWLER_TITLE } from './constants';
```

```suggestion
        <SetPageChrome trail={getEngineBreadcrumbs([CRAWLER_TITLE])} />
```

Reply:
I've fixed this in 0af45d9